recipes/doubtlab/meta.yaml

```diff
@@ -0,0 +1,65 @@
+{% set name = "doubtlab" %}
+{% set version = "0.1.1" %}
+
+
+##----------------------------------------------------------------------##
+## NOTES for Maintainers: 
+##
+## - See "maintainers_notes.md" for more details.
+##----------------------------------------------------------------------##
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: e2065c916820ac6008883032594c14a0a660862cb012f3fac152a97f4c3c64b4
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
+  run:
+    - python >=3.6
+    - cleanlab >=1.0
+    - pandas >=1.3.3    
+    - scikit-learn >=1.0.0
+    # TODO: 
+    # - remove tqdm after cleanlab-feedstock (CLFS) PR#3 is merged.
+    - tqdm 
```

Comment:
```suggestion
```
Has been merged https://github.com/conda-forge/cleanlab-feedstock/pull/3

Reply:
Thank you.